app/webpacker/components/Tickets/TicketComments.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Comment,
+  CommentAuthor,
+  CommentAvatar,
+  CommentContent,
+  CommentGroup,
+  CommentMetadata,
+  CommentText,
+  Header,
+} from 'semantic-ui-react';
+import { useQuery } from '@tanstack/react-query';
+import { DateTime } from 'luxon';
+import TicketCommentCreate from './TicketCommentCreate';
+import getComments from './api/getComments';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+import '../../stylesheets/semantic/components/comment.min.css';
+
+export default function TicketComments({ ticketId, currentStakeholder }) {
+  const [createComment, setCreateComment] = useState(false);
+
+  const {
+    data: comments, isLoading, isError, refetch, isRefetching,
```

Comment:
Why do you have to export `isRefetching` with the **re** part specifically? Shouldn't `isFetching` as a replacement for `isLoading` and `isRefetching` be good?

Reply:
See https://tanstack.com/query/latest/docs/framework/react/reference/useQuery for further information on the meaning of the different `is...` flags